RNTester/Podfile.lock

```diff
@@ -2,6 +2,11 @@ PODS:
   - boost-for-react-native (1.63.0)
   - DoubleConversion (1.1.6)
   - Folly (2018.10.22.00):
+    - boost-for-react-native
+    - DoubleConversion
+    - Folly/Default (= 2018.10.22.00)
+    - glog
+  - Folly/Default (2018.10.22.00):
```

Comment:
was this an extra alias? I don't see it used

Reply:
This is a Default subspec, so that Folly doesn't install the fabric subspec automatically as well. It is a no-op technically.

Reply:
I can make it so that it's not a no-op but actually controls what is built. I decided against this originally to prevent any mistakes from developers.

Reply:
It is better to keep default as a no-op, so user's can't accidentally forget to install Folly normally.